### ISSUE TITLE
Incorrect dependency in package.json

### ISSUE THREAD

**Issue by dinesh-chander** [keycloak/keycloak-nodejs-connect#307]
### Describe the bug

In the package.json file following dependency is defined with Github source

"keycloak-admin-client": "bucharest-gold/keycloak-admin-client"

but this repository is either not present or not public. This is breaking the example.

### Version

16.0.0-dev

**Comment by anwajler** [comment]
@dinesh-chander Maybe it should be `keycloak/keycloak-nodejs-admin-client`? Anyway, I removed this dependency and then the project started

**Comment by dinesh-chander** [comment]
yes, it should be `keycloak/keycloak-nodejs-admin-client`.

**Comment by stianst** [comment]
Tried that in #309 but doesn't work for some reason

**Comment by stianst** [comment]
@abstractj we need to get this fixed before Keycloak 17, as this is causing issues when releasing, and I've had to apply workarounds in the meantime

**Comment by abstractj** [comment]
@stianst on it.

**Comment by abstractj** [comment]
@stianst switching to the new API breaks of backwards compatibility, but we have to. Considering that I'm not familiar with the changes https://github.com/keycloak/keycloak-nodejs-admin-client and our schedule, I'd suggest to temporarily use https://github.com/keycloak/keycloak-nodejs-connect/pull/313 and revisit the refactor next year, so we can get some help from team UI.

I'm aware that this is not the best solution, but keycloak-admin-client is only used for testing purposes and does not have an impact into the codebase. //cc @jonkoops @edewit @agagancarczyk

**Comment by jonkoops** [comment]
Yeah, the new admin client is in no way compatible with what came before so we cannot simply swap it out. I agree with @abstractj that we need to work around this, but perhaps we can use the version published on NPM instead of a Github repo (see https://github.com/keycloak/keycloak-nodejs-connect/pull/313#discussion_r773449164).
